eq_translations/schema_translation.py

```diff
@@ -11,39 +11,28 @@ def __init__(self, catalog=None):
         self.catalog = catalog or self.catalog
 
     def load(self, translation_file_path):
-        with open(translation_file_path, encoding="utf8") as translation_file:
-            self.catalog = pofile.read_po(translation_file)
+        with open(
+            translation_file_path, encoding="utf8"
+        ) as translation_file:  # pragma: no cover
+            self.catalog = pofile.read_po(translation_file)  # pragma: no cover
 
     def save(self, translation_file_path):
-        with open(translation_file_path, "w+b") as translation_file:
-            pofile.write_po(translation_file, self.catalog)
-
-    @staticmethod
-    def get_comment_answer_ids(comments):
-        return [
-            comment.split(":")[1].strip()
-            for comment in comments
-            if "answer-id" in comment
-        ]
-
-    def translate_message(
-        self, message_to_translate, answer_id=None, message_context=None
-    ):
-        for message in self.catalog:
-            if message.id and are_dumb_strings_equal(message.id, message_to_translate):
-                found = True
-                comment_answer_ids = []
+        with open(translation_file_path, "w+b") as translation_file:  # pragma: no cover
+            pofile.write_po(translation_file, self.catalog)  # pragma: no cover
 
-                if message.auto_comments:
-                    comment_answer_ids = SchemaTranslation.get_comment_answer_ids(
-                        message.auto_comments
-                    )
+    @property
+    def language(self):
+        if self.catalog and self.catalog.locale:
+            return self.catalog.locale.language
 
-                if answer_id or comment_answer_ids:
-                    found = answer_id in comment_answer_ids
+    @property
+    def plural_forms(self):
+        if self.language:
+            return get_plural_forms_for_language(self.language)
```

Comment:
This property doesn't make much sense here, as the forms this returns aren't in the format that the translations file uses. I think it would be better to call the `get_plural_forms_for_language` method directly in place of a call to this method, passing it the language.